### DISCUSSION TITLE
trap invalid opcode

### DISCUSSION THREAD

**Discussion by cpcamacho954** [jedisct1/libsodium#1207] [Q&A]
Hello,

We are seeing this error in our logs
`kernel: [4155635.559765] traps: php[445083] trap invalid opcode ip:7f250f1459ff sp:7ffecd912ae8 error:0 in libsodium.so.23.3.0[7f250f121000+39000]`

Our system: 

`Package: libsodium23
Version: 1.0.18-1`

`Distributor ID: Debian
Description:    Debian GNU/Linux 11 (bullseye)
Release:        11
Codename:       bullseye
5.10.0-10-amd64 #1 SMP Debian 5.10.84-1 (2021-12-08) x86_64 GNU/Linux`

Any thoughts? Thanks in advance

**Comment by jedisct1** [comment]
Unfortunately, this is not enough information to have a clue about what could be happening.

Do you have a minimal PHP code snippet that would reliably reproduce this on a vanilla Debian installation?
